stackit/internal/services/argus/argus_acc_test.go

```diff
@@ -338,7 +363,7 @@ func TestAccResource(t *testing.T) {
 					utils.Ptr(instanceResource["metrics_retention_days"]),
 					utils.Ptr(instanceResource["metrics_retention_days_1h_downsampling"]),
 					utils.Ptr(instanceResource["metrics_retention_days_5m_downsampling"]),
-					buildAlertConfig(buildAlertConfigReceivers(true, true, true), buildAlertConfigRoute(), buildAlertConfigGlobal(true)),
+					buildAlertConfig(buildAlertConfigReceivers(true, true, true), buildAlertConfigRoute(true), buildAlertConfigGlobal(true)),
```

Comment:
We could add to the comment that we're now also adding child routes

Reply:
`// Update with Alert Configs emailConfigs and global options. Also add child routes` or something